__tests__/src/screens/NationsScreen/NationsListScreen/__snapshots__/index.js.snap

```diff
@@ -59,14 +59,18 @@ ShallowWrapper {
           }
```

Comment:
Why are the changes in snapshot? Did you change version of something?

Reply:
It happened when I merged `develop` on the branch. I didn't changed anything there, but the CI was failing so I just updated the snapshot that was needed

Reply:
Hmh, ok, strange